docs/reference/concepts/deployments.md

```diff
@@ -1,81 +1,51 @@
+For more information, see the [comprehensive launch configuration documentation](https://docs.improbable.io/reference/latest/shared/project-layout/launch-config#launch-configuration-file).
 
-### Build and upload the game assembly
+> **Note:** If the SpatialOS Runtime fails to start a worker in a deployment that you were expecting, double check that your load balancing configuration is correct.
 
-The assembly includes executable files for the client-workers and server-workers, as well as the assets both types of workers use. (The assets might be the models and textures that the client-servers - that is, the game executable code - use to visualise the game.)
+### Worker configuration
 
-  1. To build an assembly; in your Unity Editor, select **SpatialOS** > **Build all workers for cloud**.
-  1. To upload an assembly; open a terminal and navigate to the directory your game is in (the repository you’ve cloned). Run `spatial cloud upload <assembly name>`. The `<assembly name>` is a label you create so you can identify this assembly in the next step - for example you could call it `MyGDKAssembly`.
+The worker configuration file specifies the parameters of a specific worker-type as well as describing how the SpatialOS Runtime should start them. 
 
-  > **It’s finished uploading when:** You see a successful upload report printed in your terminal output, and your automatically opens [https://console.improbable.io](https://console.improbable.io) displaying the deployment.
+For more information, see the [comprehensive worker configuration documentation](https://docs.improbable.io/reference/latest/shared/project-layout/introduction#configuration-file).
 
-### Launch a cloud deployment
+> **Note:** The `build` section of the worker configuration files are **not** used by the GDK for Unity.
 
-1. Launch a cloud deployment
+## Local deployment
 
-   * In the same terminal window, run `spatial cloud launch <assembly name> cloud_launch.json <deployment name> --snapshot=snapshots/default.snapshot`
-   * This command defaults to deploying to clusters located in the US. If you’re in Europe, add the `--cluster_region=eu` flag for lower latency.
-   * It’s done when you see `Deployment launched successfully` printed in your terminal output. This might take a couple of minutes.
+To start a local deployment, in your Unity Editor, select **SpatialOS** > **Local Launch**.
 
-**About the `spatial cloud launch` command**   
-`spatial cloud launch` deploys a project to the cloud. Its full syntax is:
+This starts a local version of the SpatialOS Runtime on your development machine together with all server-workers specified in the launch configuration you used.
 
-```
-spatial cloud launch <assembly name> <launch configuration> <deployment name> --snapshot=<snapshot file>
-```
+<%(Callout message="Its done when you see the following message in the console window which opens:<br/><br/>```SpatialOS ready. Access the Inspector at https://localhost:21000/inspector```")%>
```

Comment:
should mention that you can only have one spos running locally at a time

Reply:
Also mention the error you will see if you have one running still. For easy debugging.